handbook/engineering/web/index.md

```diff
@@ -1,7 +1,7 @@
-<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 35%; float: right; margin-left: 1rem">
-
 # Web team
 
+<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 15%; float: left; margin-right: 1rem; margin-bottom: 1rem">
```

Comment:
I see you also made it smaller and floated to the left again, was that intentional?